app/models/agreement_letter.rb

```diff
@@ -90,4 +95,13 @@ def too_big?(file)
     def wrong_filetype?(file)
       file.content_type != ALLOWED_MIMETYPE
     end
+
+  def wrong_file_trailer? file
+    begin
+    page_analysis = PDF::Inspector::Page.analyze_file(file.open)
```

Comment:
This variable is not needed.
Apparently the `.open` is not needed as well, at least the tests succeed without it.